cts/lab/CTStests.py

```diff
@@ -42,120 +42,6 @@
-        ret = self.stopall(None)
-        if not ret:
-            return self.failure("Stop all nodes failed")
-
-        return self.success()
-
-    def find_leaks(self):
-        # Check for leaks
-        # (no longer used but kept in case feature is restored)
-        leaked = []
-        self._stop = StopTest(self._cm)
-
-        for node in self._env["nodes"]:
-            rc = self._stop(node)
-            if not rc:
-                self.failure("Couldn't shut down %s" % node)
-
-            (rc, _) = self._rsh(node, "grep -e indirectly.*lost:.*[1-9] -e definitely.*lost:.*[1-9] -e (ERROR|error).*SUMMARY:.*[1-9].*errors %s" % self._logger.logPat)
-            if rc != 1:
-                leaked.append(node)
-                self.failure("Valgrind errors detected on %s" % node)
-                (_, output) = self._rsh(node, "grep -e lost: -e SUMMARY: %s" % self._logger.logPat, verbose=1)
-                for line in output:
-                    self._logger.log(line)
-                (_, output) = self._rsh(node, "cat %s" % self._logger.logPat, verbose=1)
-                for line in output:
-                    self.debug(line)
-
-        self._rsh(node, "rm -f %s" % self._logger.logPat, verbose=1)
-        return leaked
-
-    def __call__(self, node):
-        #leaked = self.find_leaks()
-        #if len(leaked) > 0:
-        #    return self.failure("Nodes %s leaked" % repr(leaked))
-
-        return self.success()
-
-    @property
-    def errors_to_ignore(self):
-        """ Return list of errors which should be ignored """
-
-        return [ r"pacemaker-based.*: \*\*\*\*\*\*\*\*\*\*\*\*\*",
-                 r"pacemaker-based.*: .* avoid confusing Valgrind",
-                 r"HA_VALGRIND_ENABLED" ]
-
-
-class StandbyLoopTest(ValgrindTest):
-    '''Check for memory leaks by putting a node in and out of standby for an hour'''
-    # @TODO This is not a useful test for memory leaks
-    def __init__(self, cm):
-        ValgrindTest.__init__(self,cm)
-        self.name = "StandbyLoop"
-
-    def __call__(self, node):
-
-        lpc = 0
-        delay = 2
-        failed = 0
-        done = time.time() + self._env["loop-minutes"] * 60
-        while time.time() <= done and not failed:
-            lpc = lpc + 1
-
-            time.sleep(delay)
-            if not self._cm.SetStandbyMode(node, "on"):
-                self.failure("can't set node %s to standby mode" % node)
-                failed = lpc
-
-            time.sleep(delay)
-            if not self._cm.SetStandbyMode(node, "off"):
-                self.failure("can't set node %s to active mode" % node)
-                failed = lpc
-
-        leaked = self.find_leaks()
-        if failed:
-            return self.failure("Iteration %d failed" % failed)
-        elif len(leaked) > 0:
-            return self.failure("Nodes %s leaked" % repr(leaked))
-
-        return self.success()
-
-#AllTestClasses.append(StandbyLoopTest)
-
-
```

Comment:
Yeah it would be nice to have some valgrind capability, but it should run the usual tests under valgrind rather than be a separate test. Not worth any effort at this point.